.cspell/project-words.txt

```diff
@@ -47,3 +47,8 @@ unmock
 unstub
 unthunk
 usagestop
+PRIN
+TRAC
+DEBU
+ERRO
```

Comment:
nit: shouldn't need these anymore 🙃

Reply:
but what if, you know, we add em back in again 🙈